src/interfaces/index.ts

```diff
@@ -2,16 +2,16 @@
 // NB: this is how to re-export a type in TS
 // https://devblogs.microsoft.com/typescript/announcing-typescript-3-8-beta/
 import type { CorpInfoIF } from '@bcrs-shared-components/corp-type-module'
-import type { AlterationIF, ContactPointIF, NameRequestIF, NameTranslationIF, NameRequestDetailsIF,
-  NameRequestApplicantIF, ShareStructureIF, CourtOrderIF, IncorporationApplicationIF,
-  IncorporationAddressIf, ShareClassIF, CommentIF, ConfirmDialogType, StaffPaymentIF, ApiDateTimeUtc,
-  FormattedDateTimeGmt, IsoDatePacific } from '@bcrs-shared-components/interfaces'
+import type { AlterationIF, ChangeOfNameIF, ContactPointIF, NameRequestIF,
+  NameTranslationIF, NameRequestDetailsIF, NameRequestApplicantIF, ShareStructureIF, CourtOrderIF,
+  IncorporationApplicationIF, IncorporationAddressIf, ShareClassIF, CommentIF, ConfirmDialogType, StaffPaymentIF,
+  ApiDateTimeUtc, FormattedDateTimeGmt, IsoDatePacific, SpecialResolutionIF } from '@bcrs-shared-components/interfaces'
```

Comment:
Did you move any of these interfaces from this project to bcrs-shared-components? (And if so, can they be deleted from this project now?)

Reply:
I don't think so, mostly from `business-edit-ui`

Reply:
I honestly started refactoring as much as I could.. but realized I didn't want to go too crazy on the first time working on these repos. 

Also you probably would have told me to create a separate PR for that work.

Reply:
Sounds good. Thanks for improving this project!